core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/Invoker.scala

```diff
@@ -227,9 +231,3 @@ trait InvokerServerProvider extends Spi {
   def instance(
     invoker: InvokerCore)(implicit ec: ExecutionContext, actorSystem: ActorSystem, logger: Logging): BasicRasService
 }
-
-object DefaultInvokerServer extends InvokerServerProvider {
```

Comment:
I moved DefaultInvokerServer to a separate file， keep the same as FPCInvokerServer.